packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -406,7 +443,7 @@ export function getValidatorApi({
         slot,
         parentBlockRoot,
         randaoReveal,
-        graffiti: toGraffitiBuffer(graffiti || ""),
+        graffiti: toGraffitiBuffer(graffiti ?? defaultGraffiti),
```

Comment:
something we could consider now is applying an empty graffiti if beacon node is set to private
https://github.com/ChainSafe/lodestar/blob/b453b37d53a946b6faf73277fe1d75becff3ae8f/packages/cli/src/cmds/beacon/options.ts#L127-L129

Likely still wanna respect graffiti supplifed by validator client via api